scripts/index.js

```diff
@@ -0,0 +1,80 @@
+state = {
+    'temp': 72,
+}
+
+input.oninput = () => {
+    currentCity.innerHTML = input.value;
+};
+
+const increaseTemp = () => {
+    state.temp += 1;
+    temp.innerHTML = state.temp;
+    changeTempColor();
+    displayLandscape();
+}
+
+const decreaseTemp = () => {
+    state.temp -= 1;
+    temp.innerHTML = state.temp;
+    changeTempColor();
+    displayLandscape();
+}
+
+upArrow.addEventListener('click', increaseTemp)
```

Comment:
I would suggest putting all the event listeners in a registerEventHandlers function and then call them when the DOM content is loaded `document.addEventListener("DOMContentLoaded", registerEventHandlers)`